ml-agents/mlagents/tf_utils/globals.py

```diff
@@ -0,0 +1,13 @@
+try:
+    import horovod.tensorflow as hvd
```

Comment:
I don't think we should try to import horovod anywhere. Something like

``` python
_rank: Optional[int] = None
_local_rank: Optional[int] = None

def get_rank() -> Optional[int]:
  return _rank
```
and maybe
``` python
def set_rank(new_rank: Optional[int]) -> None:
  if _rank is not None and new_rank is not None:
    raise RuntimeError("Rank has already been set")
  _rank = new_rank
```
We can call set_rank (or just assign _rank directly) either from the distributed code, or provide an initializer via plugin.

Reply:
we don't need a set rank as horovod will assign rank to the workers and that should not be overridden